worker/transfer.go

```diff
@@ -128,7 +123,7 @@ func parallelUploadSlab(ctx context.Context, shards [][]byte, hosts []sectorStor
 			errs = append(errs, &HostError{resp.req.host.PublicKey(), resp.err})
 			// try next host
 			if hostIndex < len(hosts) {
-				reqChan <- req{hosts[hostIndex], resp.req.shardIndex}
+				go worker(req{hosts[hostIndex], resp.req.shardIndex})
```

Comment:
Spawning one goroutine per worker keeps the best case the same and gets rid of a channel which simplifies things a bit. 
It also fixes the edge case where a majority of workers are slow and we get slowed down instead of launching more goroutines for each replacement worker.

Reply:
Works for me but I think the cap on the amount of goroutines this spins up was by design. At this point the cap was meaningless anyway because we don't have any resource management of any kind.